pkg/sql/schemachanger/schemachanger_test.go

```diff
@@ -913,13 +913,33 @@ func TestComparatorFromLogicTests(t *testing.T) {
 	for _, entry := range corpus.Entries {
 		subtestName := entry.Name
 		subtestStatements := entry.Statements
-		t.Run(entry.Name, func(t *testing.T) {
+
+		if shouldSkipLogicTestCorpusEntry(subtestName) {
+			continue
+		}
+
+		if !t.Run(entry.Name, func(t *testing.T) {
 			t.Logf("running schema changer comparator testing on statements collected from logic test %q\n", subtestName)
 			ss := &staticSQLStmtLineProvider{
 				stmts: subtestStatements,
 			}
 			sctest.CompareLegacyAndDeclarative(t, ss)
 			t.Logf("schema changer comparator testing succeeded on logic test %q\n", subtestName)
-		})
+		}) {
+			t.Logf("test failed on %q\n", subtestName)
```

Comment:
I'm a bit confused about this log statement. Won't we already have the information about what test failed from `go test`s output?

Reply:
I am having a bit trouble understanding the TC behavior as well. See this ongoing slack [thread](https://cockroachlabs.slack.com/archives/CJ0H8Q97C/p1696862216857819). I think we don't need it -- it's just this `crdb_internal` test case is giving us some extra hard time. 

Let me remove them and see.